src/main/java/org/dbos/apiary/xa/benchmarks/XABankBenchmark.java

```diff
@@ -36,12 +39,40 @@ public class XABankBenchmark {
     private static final Collection<Long> auditTimes = new ConcurrentLinkedQueue<>();
     private static final Collection<Long> transferTimes = new ConcurrentLinkedQueue<>();
 
-    public static void benchmark(String mainHostAddr, String postgresAddr, String mysqlAddr, Integer interval, Integer duration, int percentageTransfer) throws SQLException, InterruptedException {
-        MySQLXAConnection mysqlConn = new MySQLXAConnection(mysqlAddr, XAConfig.postgresPort, "dbos", "root", "dbos");
-        PostgresXAConnection postgresConn = new PostgresXAConnection(postgresAddr, XAConfig.mysqlPort, "dbos", "postgres", "dbos");
+    static XAConnection getBitronix2MySQLXAConnection(String mysqlAddr) throws SQLException {
+        BitronixXADBConnection mysqlConn = new BitronixXADBConnection("MySQL" + UUID.randomUUID().toString(), "com.mysql.cj.jdbc.MysqlXADataSource", mysqlAddr, XAConfig.mysqlPort, "dbos", "root", "dbos");
+        BitronixXADBConnection mysql2Conn = new BitronixXADBConnection("MySQL" + UUID.randomUUID().toString(), "com.mysql.cj.jdbc.MysqlXADataSource", mysqlAddr, XAConfig.mysql2Port, "dbos", "root", "dbos");
+        // Hack: Use mysqlConn as postgres connection to get a setup with 2 mysql instances.
+        XAConnection conn = new BitronixXAConnection(mysqlConn, mysql2Conn);
+        return conn;
+    }
+
+    static XAConnection getBitronixXAConnection(String postgresAddr, String mysqlAddr) throws SQLException {
+        BitronixXADBConnection mysqlConn = new BitronixXADBConnection("MySQL" + UUID.randomUUID().toString(), "com.mysql.cj.jdbc.MysqlXADataSource", mysqlAddr, XAConfig.mysqlPort, "dbos", "root", "dbos");
+        BitronixXADBConnection postgresConn = new BitronixXADBConnection("Postgres" + UUID.randomUUID().toString(), "org.postgresql.xa.PGXADataSource", postgresAddr, XAConfig.postgresPort, "dbos", "postgres", "dbos");
+        XAConnection conn = new BitronixXAConnection(postgresConn, mysqlConn);
+        return conn;
+    }
+
+    static XAConnection getXJXAConnection(String postgresAddr, String mysqlAddr) throws SQLException {
```

Comment:
It would be better to have more intuitive names such as "getPostgresMysqlConnection" here :)